Smarty/templates/LoginHeader.tpl

```diff
@@ -14,7 +14,7 @@
 <meta http-equiv="Content-Type" content="text/html; charset={$LBL_CHARSET}">
 <meta http-equiv="X-FRAME-OPTIONS" content="DENY" />
 <title>{$APP.LBL_BROWSER_TITLE}</title>
-<link REL="SHORTCUT ICON" HREF="themes/images/favicon.ico">
+<link REL="SHORTCUT ICON" HREF="{php} $logo_login=fetch_logo(3); if ($logo_login!='test/logo/') echo $logo_login; else echo 'themes/images/favicon.ico' {/php}">
```

Comment:
I changed the code. Couldn't use header.php for login variables. Used modules/Users/Login.php